shared/unattended/Fedora-24.ks

```diff
@@ -0,0 +1,40 @@
+install
+KVM_TEST_MEDIUM
+text
```

Comment:
This should be `GRAPHICAL_OR_TEXT` as we should test both, graphic and text version...